scripts/bootstrap/07-deploy-concourse.sh

```diff
@@ -5,14 +5,15 @@ set -eux
 # Set deploy-concourse pipeline
 fly --target bootstrap set-pipeline \
   --pipeline deploy-concourse \
-  --config ../cg-deploy-concourse/ci/pipeline-development.yml \
+  --config ../cg-deploy-concourse/ci/pipeline.yml \
   --load-vars-from ${WORKSPACE_DIR}/concourse-environment.yml \
   --load-vars-from ../cg-deploy-concourse/ci/concourse-defaults.yml \
   --var secrets-bucket=${VARZ_BUCKET} \
   --var concourse-production-deployment-bosh-target=$(bosh interpolate ${WORKSPACE_DIR}/tooling-state.yml --path /terraform_outputs/tooling_bosh_static_ip) \
   --var concourse-production-private-passphrase=${CONCOURSE_SECRETS_PASSPHRASE} \
   --var tf-state-bucket=${TF_STATE_BUCKET} \
-  --var slack-webhook-url=${SLACK_WEBHOOK_URL}
+  --var slack-webhook-url=${SLACK_WEBHOOK_URL} \
+  --ops-file bosh/opsfiles/development.yml
```

Comment:
Does this work? I tried passing `--ops-file` to `fly set-pipeline` and got ```error: unknown flag `ops-file'```.

Reply:
Hmm.  You are right.  I will get a bosh int in there.  I haven't worked myself back to where I can use this yet.  I plan on rebuilding the whole env from scratch, but wanted to work through the issues that I could work through with the existing setup first.

Thanks!